eahub/base/static/global/styles/components/_feedback.scss

```diff
@@ -0,0 +1,35 @@
+@media (min-width: 601px) {
+  .feedbackbtn {
+    top: 60%;
+  }
+}
+
+@media (max-width: 600px) {
+  .feedbackbtn {
+    bottom: -30px; 
+  }
+}
+
+.feedbackbtn {
+  position: fixed;
+  transform: rotate(-90deg);
+  transform-origin: top left;
+  color: #fff;
+  text-align: center;
+  right: -65px;
+} 
+
+a {
+  color: inherit;
+}
```

Comment:
those would be global styles? I'm not sure how all browsers would process that

could we encapsulate it in a component class?

`<div class=feedback-componenent>`

```
.feedback-componenent {
    // other styles
}
```